src/components/App.tsx

```diff
@@ -6201,5 +6225,30 @@ if (
       },
     },
   });
+
+  if (process.env.NODE_ENV === ENV.TEST) {
+    // Taken from https://github.com/tc39/proposal-relative-indexing-method#polyfill so that it works in tests
+    /* eslint-disable */
+    Object.defineProperty(Array.prototype, "at", {
```

Comment:
`.at()` support [isn't quite there yet](https://caniuse.com/mdn-javascript_builtins_array_at), so either add this polyfill to all environments (and put this into index somewhere so it's defined for the whole app), or don't use it.

Reply:
create a polyfil file and imported so its available in all environements